CHANGELOG.md

```diff
@@ -25,42 +27,28 @@ All notable changes to this project will be documented in this file.
 - Prevent implicit transactions from spanning multiple chunks [EventStore#3918](https://github.com/EventStore/EventStore/pull/3918)
 - Documentation for FIPS 140-2 compliance [EventStore#3948](https://github.com/EventStore/EventStore/pull/3948)
 - Documentation for redaction [EventStore#3949](https://github.com/EventStore/EventStore/pull/3949)
-- Add steps to update certificates [EventStore#3940](https://github.com/EventStore/EventStore/pull/3940)
+- Add steps in Documentation to update certificates [EventStore#3940](https://github.com/EventStore/EventStore/pull/3940)
 - Call home database telemetry. [EventStore#3947](https://github.com/EventStore/EventStore/pull/3947)
 - Support for more plugins use cases [EventStore#3984](https://github.com/EventStore/EventStore/pull/3984)
 - Support for pluggable subsystems [EventStore#3986](https://github.com/EventStore/EventStore/pull/3986)
 - Support clusters with nodes that have certificates with different CNs [EventStore#3960](https://github.com/EventStore/EventStore/pull/3960)
 
 ### Fixed
-- PersistentConfig1 system event does not begin with $ [EventStore#3932](https://github.com/EventStore/EventStore/pull/3932)
 - race conditions when caching/uncaching chunks [EventStore#3930](https://github.com/EventStore/EventStore/pull/3930)
 - Certain configuration parameters (mainly Interface options) have been renamed, which would make it more clear as to what each parameter/option does. [EventStore#3907](https://github.com/EventStore/EventStore/pull/3907)
```

Comment:
this probably is 'changed' rather than fixed as well. i wonder if we should introduce a breaking changes section